frontend-react/src/components/Content/PageGenerationTools.tsx

```diff
@@ -5,12 +5,12 @@ import { CrumbConfig, WithCrumbs } from "../Crumbs";
 import { ContentDirectory } from "./MarkdownDirectory";
 
 export const contentContainer = (
-    content: () => JSX.Element,
+    content: JSX.Element,
```

Comment:
SMALLLLLLL POINT, but I think one thing we could do is make this an `FC` type and render it in the JSX below:

```jsx
<div className="grid-container rs-documentation usa-prose">
   <content />
</div>
```

We wouldn't need to eager-render [below](https://github.com/CDCgov/prime-reportstream/pull/7881/files#diff-90b710ea5830219761e9f4830e9e7cba7ce017b1daa0510799773784495d770bR78), and we could just continue to pass the component reference rather than the rendered component.

Reply:
Much prefer this, actually! I'd stepped on my own toes when I changed the parameter type from a function to the element itself. Patched it up, cherry picked it on my other branch where I found the bug, and it worked.

Reply:
Yay!  Glad it worked out 🎉